mdata/aggmetric.go

```diff
@@ -58,8 +58,8 @@ func NewAggMetric(store Store, cachePusher cache.CachePusher, key schema.AMKey,
 
 	ingestFromT0 := uint32(0)
 	if ingestFrom > 0 {
-		// compute start of next chunk after ingestFrom
-		ingestFromT0 = uint32(ingestFrom-(ingestFrom%int64(ret.ChunkSpan))) + ret.ChunkSpan
+		// compute next chunk boundary starting from 'ingestFrom'
```

Comment:
the word "next" seems confusing here.

Reply:
> the word "next" seems confusing here.

Yeah, I don't know how to phrase it much better :/ And `AggBoundary` uses the word next too :/
`// AggBoundary returns ts if it is a boundary, or the next boundary otherwise.`

Should I remove the comment altogether?

Reply:
what i meant is that "next" does not apply in all cases.  AggBoundary comment has the appropriate nuance. anyway i'll give it a shot, no worries.

Reply:
> what i meant is that "next" does not apply in all cases. AggBoundary comment has the appropriate nuance. anyway i'll give it a shot, no worries.

you're right